index.html

```diff
@@ -8,13 +9,116 @@
+            <div class="new-button-row">
+              <button class="like-button"><i class="fa-regular fa-heart"></i></button>
+            </div>
+          </div>
+          <div class="row img-placeholder">
+            <div class="column-full">
+              <img src="images/depositphotos_137014128-stock-illustration-user-profile-icon.jpg" alt="placeholder" />
+            </div>
+          </div>
+          <div class="column-half information-details">
+            <p class="born">
+              <span class="info-header">BORN</span>
+              <span class="born-info"></span>
+            </p>
+            <p class="died">
+              <span class="info-header">DIED</span>
+              <span class="died-info"></span>
+            </p>
+            <p class="blood">
+              <span class="info-header">BLOOD STATUS</span>
+              <span class="blood-status-info"></span>
+            </p>
+            <p class="house">
+              <span class="info-header">HOUSE</span>
+              <span class="house-info"></span>
+            </p>
+            <p class="patronus">
+              <span class="info-header">PATRONUS</span>
+              <span class="patronus-info"></span>
+            </p>
+            <p class="wand">
+              <span class="info-header">WAND</span>
+              <span class="wand-info"></span>
+            </p>
+            <p class="hp-wiki">
+              <span class="info-header">HARRY POTTER WIKI</span>
+              <span class="hp-info"></span>
+            </p>
+          </div>
+        </div>
+      </div>
+    </div>
   </header>
+
   <main>
+    <div class="character-container"></div>
+    <div class="favorite-containter hidden">
+      <div data-view="favorite-view" class="fave-view">
+        <div class="row">
+          <div class="column-one-fourth-left">
+            <button class="x-button"><i class="fa-solid fa-circle-xmark"></i></button>
+          </div>
+          <div class="column-one-third nav-bar">
+            <h1>FAVORITES</h1>
+          </div>
+          <div class="column-one-fourth">
+            <button class="red-favorite-page"><i class="fa-solid fa-heart"></i></button>
+          </div>
+        </div>
+        <div class="row">
+          <div class="column-full">
+            <p class="no-favorites hidden">There are no favorites saved.</p>
+          </div>
+        </div>
+        <div class="column-full">
+          <div class="row button-row">
+            <a class="delete-entry visibility-hidden">Delete</a>
+          </div>
+        </div>
+        <div class="modal-container hidden">
+          <div class="modal-content">
+            <h3>Are you sure you want to delete this?</h3>
+            <div class="row button-row">
+              <button class="button-cancel">CANCEL</button>
+              <button class="button-confirm">CONFIRM</button>
+            </div>
+          </div>
+        </div>
+      </div>
```

Comment:
Same with this. Where is this and when does it appear? If it has no functionality yet, then it shouldn't be included in this feature